### DISCUSSION TITLE
High percentage of alignment failed due to no hits (not TCR/IG?) in 10X TCR data

### DISCUSSION THREAD

**Discussion by genome2020** [milaboratory/mixcr#1754] [Q&A] [answered]
Hi 
I have analyzed my 10X TCR repertoire data using the 'analyze' command in MiXCR using the '10x-sc-xcr-vdj' preset. However, I am getting a significant percentage of reads (~15-50%) failing alignment due to 'no hits (not TCR/IG?)'. I aligned the unaligned reads against the hg38 genome and I see a good percentage of these reads mapped to the TCR genes loci. Would you have any suggestions on how I can improve my alignment rate and recover the unaligned reads? 

I am using MiXCR version 4.6.0. 

Here is the command I am using

mixcr -Xmx100g analyze 10x-sc-xcr-vdj --species hsa sample1_R1_001.fastq.gz" smaple2_R2_001.fastq.gz" sample1

**Comment by mizraelson** [comment]
Hi, the command is correct. This sometimes happens if you have many reads covering only the C gene. MiXCR reports such reads as non-target because it looks for reads that at least partially cover the V, J, or D genes. If you can share the data, I can take a look and confirm. You can send it to support@milaboratories.com.

**Comment by genome2020** [comment]
Hi @mizraelson 
Thanks! What data would you need?

**Reply by mizraelson** [reply]
Raw FASTQ files

**Reply by mizraelson** [reply, marked as answer]
I have reviewed the data. I used the latest MiXCR v4.7, which works significantly better for 10x data, and the alignment rate is now 71%. The issue originally was with a high rate of reads that only covered the C gene. Such reads are now assembled into longer consensuses prior to alignment. How many cells do you expect for this sample?

**Reply by genome2020** [reply]
Thanks @mizraelson for checking. I will switch to the newer version. We initially loaded 5000 cells.
